front/app/components/IdeaCards/IdeasWithFiltersSidebar/index.tsx

```diff
@@ -202,6 +264,19 @@ const IdeaCards = ({ ideaQueryParameters, onUpdateQuery }: Props) => {
     setFiltersModalOpened(false);
   }, []);
 
+  const [isCTABarVisible, setIsCTABarVisible] = useState(false);
+
+  useEffect(() => {
+    function checkCTABarVisibility() {
+      if (document.getElementById('project-cta-bar')) {
+        setIsCTABarVisible(true);
+      }
+    }
+
+    window.addEventListener('scrollend', checkCTABarVisibility);
+    return () => window.removeEventListener('scroll', checkCTABarVisibility);
+  }, []);
```

Comment:
There are a few issues which I've addressed in this PR: https://github.com/CitizenLabDotCo/citizenlab/pull/9498.